Src/DependencyCollector/Shared/DependencyTrackingTelemetryModule.cs

```diff
@@ -173,11 +185,11 @@ protected virtual void Dispose(bool disposing)
             {
                 if (disposing)
                 {
-#if !NET40 && !NETCORE
+#if NET45
```

Comment:
since there are only 40, 45 and CORE, those are equivalents, but I will change definition condition to be consistent